webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -300,11 +302,14 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
             : 0,
         );
 
+        const heatMap = await getProjectNumericMetricsRange({ id: project.id });
+        setProjectHeatmap(heatMap);
```

Comment:
move `setProjectHeatmap` into the `mounted` check.

Reply:
actually -- should this be in the same effect? we could split it out so we can request `getProjectColumns` and `getProjectNumericMetricsRange` in parallel.